aws_backup.tf

```diff
@@ -127,3 +135,11 @@ resource "aws_backup_vault_policy" "target" {
   backup_vault_name = aws_backup_vault.target[0].name
   policy            = data.aws_iam_policy_document.target_vault[0].json
 }
+
+resource "aws_backup_vault_lock_configuration" "target" {
+  count               = var.enabled && var.is_cross_account_backup_enabled && var.vault_lock_enabled ? 1 : 0
```

Comment:
Would it make sense to have vault lock enabled only in the target account, but not in the source one? If so, can we add support for this setup?